Dockerfile.helper

```diff
@@ -1,7 +1,11 @@
-FROM busybox:latest
+FROM alpine:latest
 
 ARG DATA_DIRECTORY=data
 
-USER 1000
+COPY --chown=1000:1000 ${DATA_DIRECTORY} /container-data
 
-COPY --chown=1000:1000 ${DATA_DIRECTORY} /container-data
\ No newline at end of file
+RUN ([ -d /container-data/sqlserver ] && addgroup --gid 10001 mssql && adduser -S -s /usr/sbin/nologin -h /home/mssql --disabled-password -G mssql -u 10001 mssql) || echo
+
+RUN ([ -d /container-data/sqlserver ] && chown -R 10001:10001 /container-data/sqlserver) || echo
```

Comment:
it would be good to parameterize the `sqlserver` directory (it's the volume mount defined in the sqlserver Dockerfile); however, i'm not sure how to go about this.

Reply:
We could parameterize it, but it's really unlikely to change since it just matches the service name. At the point we change it, we're doing wide search-and-replace anyways, so I think this is just fine for now.